### PR TITLE
Add skaffold for debugging

### PR DESCRIPTION
**Pull request by kemurayama**
When we want to modify the space-agon and try our original logics, we have to build and deploy by ourselves currently. This PR enables us to run the cluster with `skaffold` command. 

- Changed `Dockerfiles` to reduce their image size
- Created `make skaffold-setup` for `skaffold`
- Canged README.md
- Updated Open Match  to `v1.4.0`

Closes (#5)

### CODE REVIEW

File: scripts/setup-skaffold.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# limitations under the License.
+
+# DO NOT RUN this script directly.
+# Run `make skaffold-setup` command after creating the cluster.
+# This script setup skaffold.yaml
+
+PROJECT_ID=$1
+REGISTRY=$2
+
+gcloud config set project $PROJECT_ID
```
Comment (govargo) [inline scripts/setup-skaffold.sh:24]: ${PROJECT_ID} is better.

File: scripts/setup-skaffold.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+gcloud services enable cloudbuild.googleapis.com
+
+gcloud projects add-iam-policy-binding $PROJECT_ID \
+    --member=serviceAccount:$(gcloud projects describe $PROJECT_ID \
```
Comment (govargo) [inline scripts/setup-skaffold.sh:28]: ${PROJECT_ID} is better too.

Comment (govargo) [inline scripts/setup-skaffold.sh:28]: ${PROJECT_ID} is better too.

Comment (kemurayama) [reply scripts/setup-skaffold.sh]: Thanks. I added `{}`.

File: scripts/setup-skaffold.sh (hunk 1 above)
Comment (kemurayama) [reply scripts/setup-skaffold.sh]: Added `{}`

**Review by govargo — APPROVED** [review]
LGTM

**Comment by kemurayama** [comment]
Thanks !

**Pull request merged**